common/src/chain/config/regtest_options.rs

```diff
@@ -39,55 +39,55 @@ use paste::paste;
 #[derive(Args, Clone, Debug)]
 pub struct ChainConfigOptions {
     /// Magic bytes.
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_CHAIN_MAGIC_BYTES")]
     pub chain_magic_bytes: Option<String>,
 
     /// The maximum future block offset in seconds.
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_CHAIN_MAX_FUTURE_BLOCK_TIME_OFFSET")]
     pub chain_max_future_block_time_offset: Option<u64>,
 
     /// The software version (major.minor.path).
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_SOFTWARE_VERSION")]
     pub software_version: Option<String>,
 
     /// Target block spacing in seconds.
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_CHAIN_TARGET_BLOCK_SPACING")]
     pub chain_target_block_spacing: Option<u64>,
 
     /// Coin decimals.
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_CHAIN_COIN_DECIMALS")]
     pub chain_coin_decimals: Option<u8>,
 
     /// Emission schedule (`<initial_supply>+<initial_subsidy>[, <height>+<subsidy>]`).
     pub chain_emission_schedule: Option<String>,
 
     /// The maximum block header size in bytes.
-    #[clap(long)]
+    #[clap(long, env = "MINTLAYER_CHAIN_MAX_BLOCK_HEADER_SIZE")]
     pub chain_max_block_header_size: Option<usize>,
```

Comment:
I'd probably slightly prefer `MINTLAYER_REGTEST_*` here in place of `MINTLAYER_CHAIN_*`. Thoughts?

Reply:
If it's regtest-specific, then sure. But I'd rather change both names, not just the env var value.

Reply:
> If it's regtest-specific, then sure. But I'd rather change both names, not just the env var value.

Hmm, ok. Changing switch names here might be a bit of a hassle, I don't have a strong opinion about this.